src/kirby/components/list/list.component.scss

```
@@ -30,6 +30,10 @@ ion-list, .list-items {
   }
 }
 
+.list-header {
```

Just use `ion-list-header` as selector:
```suggestion
ion-list-header {
```